README.md

```diff
@@ -177,239 +189,175 @@ sergio@sergio:~/plz/examples/pytorch$ plz history | \
-so that it's ready for your instances to use*
+1. Install the AWS CLI: `pip install awscli`
+2. Configure it with your access key: `aws configure`
 
-*Note: if you usually use AWS in a particular region, please edit
-aws_config/config.json and set your region there. The default file sets the
-region to eu-west-1, Ireland.*
+If you usually use AWS in a particular region, please edit
+`aws_config/config.json` and set your region there. The default file sets the
+region to _eu-west-1_ (Ireland).
 
+Then run:
 
 ```
-# Configure access to AWS.
-# You'll need your access key. You can get it from the AWS console.
-# In the top bar, click on your name, then `My security credentials`, then
-# create one in `Access keys`
-# You only need to set the access key ID and the secret access key
-
-aws configure
-
-# Check that AWS works. You should see a possibly empty list of instances
-
-aws ec2 describe-instances --region eu-west-1
-
-# Start the AWS controller. This runs detached and spits the output to the
-# terminal. You might want to run it in a different terminal (remember
-# `cd plz/`). Remember to edit `aws_config/config.json` to set your region,
-# unless you want eu-west-1.
-
 ./start_aws_controller
 ```
 
-## Examples
+Unless you add `"instance_max_uptime_in_minutes": null,` to your
+`plz.config.json`, all AWS instances you start terminate after 60 minutes.
+That's on purpose, in case you're just trying the tool and something doesn't go
+well (for example, there's a power cut). You can always use `plz list` and `plz kill` before leaving your computer, as to make sure that there no instances
+remaining. For maximum assurance, we recommend checking the state of your
+instances in the AWS console.
 
-*Note: if you want to run the examples using the AWS instances, be aware that
-this has a cost. You can change the value of
-`"max_bid_price_in_dollars_per_hour": N` in `plz.config.json` to any value
-you like. Examples takes around 5 minutes to run.
-The value in the provided configs range from 0.5 dollars/hour to 2 dollars/hour
-(for GPU-powered machines). See the following note as well.*
+If you want to run the examples using the AWS instances, be aware that this has
+a cost. You can change the value of `"max_bid_price_in_dollars_per_hour": N` in
+`plz.config.json` to any value you like. Examples takes around 5 minutes to run.
+The value in the provided configs range from \$0.5/hour to \$2/hour (for
+GPU-powered machines).
 
-*Note: unless you add `"instance_max_uptime_in_minutes": null,` to your
-`plz.config.json`, all AWS instances you start terminate after 60 minutes.
-That's on purpose, in case you're just trying the tool and something doesn't
-go well (like, there's a power cut). You can always use `plz list` and
-`plz kill` before leaving your computer, as to make sure that there no
-instances remaining. For maximum assurance, you can check your instances in
-the AWS console.*
+## Examples
 
+### Python
```

Comment:
Don't know what to do about this python example... It's a terrible one (I put it there in a rush to have an example before the ML4P workshop). Definitely not putting it first. I was thinking of not mentioning it in the README at all, then I though that maybe it's useful to show you can run plain python, or to show something that doesn't look scary to someone who doesn't do machine learning but... Well, if you can run pytorch you already know you can run plain python, and I'm not sure whether people would get scared by looking at ML code (they'll probably just run it and ignore the details if they don't understand them) (So deleting from the README is my inclination today. Whatcha think?)

Reply:
I suggest we leave it there and see if people improve it. 😄

Reply:
I might write a better one as part of talk preparation.